docs/gatsby/api-reference/ocular-config.md

```diff
@@ -18,6 +18,8 @@
 | `EXAMPLES`         | `Array` | See below |
 | `INDEX_PAGE_URL` | `String`  | Optional URL to a replacement component for the home page. |
 | `DOC_PAGE_URL` | `String`  | Optional URL to a replacement component for doc pages. |
+| `DOC_FOLDER` | `String`  | The path to the doc folder. |
```

Comment:
right now DOC_FOLDER doesn't do anything

Reply:
But my changes will need it?

Reply:
If DOC_FOLDER doesn't do anything let's delete it and use DOC_FOLDERS only? Having two of these does not seem to be in the user's interest.